specification/resources/load_balancers/models/forwarding_rule.yml

```
@@ -13,7 +13,10 @@ properties:
     example: https
```

```suggestion
    - udp
    example: https
```

Looks like `udp` is missing from the enum list for `entry_protocol`